src/main/java/software/aws/neptune/gremlin/GremlinQueryExecutor.java

```diff
@@ -157,7 +160,20 @@ public static Cluster.Builder createClusterBuilder(final GremlinConnectionProper
             builder.minSimultaneousUsagePerConnection(properties.getMinSimultaneousUsagePerConnection());
         }
         if (properties.getAuthScheme() == AuthScheme.IAMSigV4) {
-            builder.channelizer(SigV4WebSocketChannelizer.class);
+            builder.handshakeInterceptor( r ->
```

Comment:
Duplicated code - let's extract it, to avoid copy-pasting.

Reply:
fixed in  https://github.com/aws/amazon-neptune-jdbc-driver/pull/230/commits/c2a7e780ea9304e77c7c7c010c27dae691cd260a